project_resource_calendar/tests/common.py

```diff
@@ -19,16 +24,25 @@ def setUp(self):
             'resource_type': 'room',
             'allow_double_book': True,
         })
-
         self.instrument_1 = self.Instruments.create({
             'name': 'Test Instrument 1',
             'resource_type': 'material',
             'room_id': self.room_1.id,
             'allow_double_book': True,
         })
-    
         self.weekly_report_wizard_common = self.WeeklyReportWizards.create({
                 'room_id': self.room_1.id,
                 'recurrency': True,
                 'state': 'open',
             })
+        self.room_2 = self.Rooms.create({
+            'name': room_name,
+            'resource_type': resource_type_name,
+            'allow_double_book': double_book_status,
+        })
+        self.instrument_2 = self.Instruments.create({
+            'name': instrument_name,
+            'resource_type': instrument_type_name,
```

Comment:
check the resource type it should be a material

Reply:
@rimbendhaou Done